lib/rest-adapter.js

```diff
@@ -586,7 +609,7 @@ function RestMethod(restClass, sharedMethod) {
   this.notes = sharedMethod.notes;
   this.documented = sharedMethod.documented;
 
-  var methodRoutes = getRoutes(sharedMethod);
+  var methodRoutes = getRoutes(sharedMethod, options);
```

Comment:
Although, when I look at the two places where `new RestMethod()`is called, they both receive the exact same `options` value as the `RestClass()` constructor from which they are called. The same value is passed on to `getRoutes()` also to determine the value of `this.routes` there. I don't see the `options` being stored on the `RestClass` instance: 

https://github.com/lehni/strong-remoting/blob/master/lib/rest-adapter.js#L577-L591